packages/perseus/src/widgets/image/image.class.tsx

```diff
@@ -0,0 +1,82 @@
+import {linterContextDefault} from "@khanacademy/perseus-linter";
+import * as React from "react";
+import _ from "underscore";
+
+import {PerseusI18nContext} from "../../components/i18n-context";
+import {getPromptJSON as _getPromptJSON} from "../../widget-ai-utils/image/image-ai-utils";
+
+import {ImageComponent} from "./image";
+
+import type {WidgetExports, WidgetProps, Widget} from "../../types";
+import type {ImagePromptJSON} from "../../widget-ai-utils/image/image-ai-utils";
+import type {Range, PerseusImageWidgetOptions} from "@khanacademy/perseus-core";
+
+const defaultBoxSize = 400;
+const defaultRange: Range = [0, 10];
+const defaultBackgroundImage = {
+    url: null,
+    width: 0,
+    height: 0,
+} as const;
+
+type ExternalProps = WidgetProps<PerseusImageWidgetOptions>;
+
+export type ImageProps = ExternalProps & {
+    alignment: NonNullable<ExternalProps["alignment"]>;
+    title: NonNullable<ExternalProps["title"]>;
+    range: NonNullable<ExternalProps["range"]>;
+    box: NonNullable<ExternalProps["box"]>;
+    backgroundImage: NonNullable<ExternalProps["backgroundImage"]>;
+    labels: NonNullable<ExternalProps["labels"]>;
+    alt: NonNullable<ExternalProps["alt"]>;
+    caption: NonNullable<ExternalProps["caption"]>;
+    linterContext: NonNullable<ExternalProps["linterContext"]>;
+};
+
+type DefaultProps = {
```

Comment:
```suggestion
interface DefaultProps extends Pick<
    ImageProps,
    "alignment" |
    "title" |
    "range" |
    "box" |
    "backgroundImage" |
    "labels" |
    "alt" |
    "caption" |
    "linterContext"
> {}
```
Using pick here will help avoid repetition and keeps types consistent.